clients/src/main/java/org/apache/kafka/clients/KafkaClient.java

```diff
@@ -86,11 +86,21 @@
     List<ClientResponse> poll(long timeout, long now);
 
     /**
+     * Diconnects the connection to a particular node, if there is one.
+     * Any pending ClientRequests for this connection will receive disconnections.
+     *
+     * @param nodeId The id of the node
+     */
+    void disconnect(String nodeId);
+
+    /**
      * Closes the connection to a particular node (if there is one).
+     * All requests on the connection will be cleared.  ClientRequest callbacks will not be invoked
+     * for the cleared requests, nor will they be returned from poll().
      *
      * @param nodeId The id of the node
      */
-    void close(String nodeId);
+    void forget(String nodeId);
```

Comment:
`close` implies cleaning up resources, which this does not.  Similarly, I would expect `AdminClient#close` to either complete or abort any futures that are in progress, not just to forget about them.  `FileOutputStrem#close` flushes any modifications to disk; it doesn't just drop them.

In the interests of time I will leave this as `close` and we can discuss it later.